src/MatrixDepot.jl

```diff
@@ -115,25 +117,34 @@ function init(;ignoredb::Bool=false)
         mkpath(data_dir())
     end
 
-    if !isdir(MYDEP)
-        mkpath(MYDEP)
-        open(joinpath(MYDEP, GROUP), "w") do f
-            write(f, "usermatrixclass = Dict(\n);")
+    if isdir(MYDEP) #Backward compatibility check deprecation. Delete eventually.
```

Comment:
This was resolved in https://github.com/peterahrens/MatrixDepot.jl/pull/79/commits/4b07108d4225dbcccef7deaa0ce235a617abd9f9. There was some complicated code to avoid issuing warnings for users that do not customize the old default files in `MYDEPOT`. I simplified the code here.